yjit_codegen.c

```diff
@@ -183,6 +186,26 @@ yjit_gen_exit(jitstate_t *jit, ctx_t *ctx, codeblock_t *cb)
     return code_ptr;
 }
 
+// Generate an interpreter to REG_CFP->pc.
+static uint8_t *
+yjit_gen_context_free_exit(codeblock_t *cb)
+{
```

Comment:
Hmm. Maybe this should be called `gen_return_exit`? Also would like a better comment explaining the purpose of this function.